rules/aws_cloudtrail_rules/aws_rds_master_pass_updated.yml

```diff
@@ -0,0 +1,289 @@
+AnalysisType: rule
+Description: A sensitive database operation that should be performed carefully or rarely
+DisplayName: AWS RDS Master Password Updated
+Enabled: true
+Filename: aws_rds_master_pass_updated.py
+Severity: Low
+DedupPeriodMinutes: 60
+Reports:
+    MITRE ATT&CK:
+        - TA0003:T1098
+SummaryAttributes:
+    - awsRegion
+    - userIdentity:arn
+    - responseElements:dBInstanceIdentifier
+    - p_any_aws_arns
+    - p_any_aws_account_ids
+LogTypes:
+    - AWS.CloudTrail
+RuleID: AWS.RDS.MasterPasswordUpdated
+Threshold: 1
+Tests:
+    - ExpectedResult: false
+      Log:
+        awsRegion: us-west-1
```

Comment:
I am unsure of the implications of not wrapping the log event in `{}` . We received another PR with this non-curly'ed format, and we do now know of possible downstream impact from missing the curlies. 

Additionally, the test's Name attribute is historically at top-of-test, which helps to identify when manually troubleshooting.

Reply:
There's no implication of using YML IMO, it's the same thing. FWIW, this is the result when you download from the UI.